server/src/inmantals/server.py

```diff
@@ -56,13 +57,17 @@ def __init__(self, instream: BaseIOStream, outstream: BaseIOStream, address):
         self.reverse_anchormap = None
         self.state_lock: asyncio.Lock = asyncio.Lock()
         self.diagnostics_cache: Optional[lsp_types.PublishDiagnosticsParams] = None
+        self.compiler_venv_path: Optional[str] = None
 
     async def initialize(self, rootPath, rootUri, **kwargs):  # noqa: N803
         logger.debug("Init: " + json.dumps(kwargs))
 
         self.rootPath = rootPath
         self.rootUrl = rootUri
         os.chdir(rootPath)
+        init_options = kwargs.get("initializationOptions", None)
+        if init_options:
+            self.compiler_venv_path = init_options.get("compilerVenv", None)
```

Comment:
Perhaps default to `os.path.join(self.rootPath, ".env-ls")` or something similar to support other clients than the vscode one.

Reply:
Good idea, but I'm not sure about the naming. 
`.env-ls` kind of suggests a bit that the ls is installed there
What I wrote now `.env-ls-compiler`, doesn't really make it clear either.
Any suggestions @sanderr ?

Reply:
You're right. I think `.env-ls-compiler` is sufficiently clear, though it's not very concise but I guess that doesn't really matter. On the other hand, I think 99% of people working with inmanta will either use the language server with vscode or not at all, so the naming for this isn't terribly important. I'm fine with either one, no others come to mind.